xla/backends/gpu/runtime/command_buffer_cmd.h

```diff
@@ -99,6 +99,10 @@ enum class CommandBufferCmdType : int32_t {
 #undef DECLARE_ENUM
 };
 
+// Command priority in command buffer, currently only support default, low and
+// high priority.
+enum class CommandBufferCmdPriority { kDefault, kLow, kHigh };
```

Comment:
It doesn't look like another enum is worth it, jus reuse the stream_executor once.

Reply:
Fixed.